src/test/osd/osd-fast-mark-down.sh

```diff
@@ -99,12 +98,10 @@ function test_fast_kill() {
      if [ $down_osds -lt $i ]; then
         echo Killed the OSD, yet it is not marked down
         ceph osd tree
-        teardown $dir 
         return 1
      fi
    done
    pkill -SIGTERM rados
-   teardown $dir || return 1
```

Comment:
Removing this one in particular will cause entire test to fail, even if it actually succeeded, because OSDs are not cleaned up in subsequent sub-tests.